.changeset/tricky-spiders-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
```

Comment:
I would suggest going with `minor` and also add the rest of the packages that are affected so they release a new version.
```suggestion
---
'@clerk/clerk-js': minor
'@clerk/types': minor
'@clerk/localizations': minor
---
```

Reply:
I forgot to re-run the changeset script after pushing changes on other packages, thanks for the catch here 🙇🏻